resources/Fluxes/NUMI/NUMI-v1.0/NUMI_RHC_LE.dat

```diff
@@ -1,4 +1,4 @@
-#Enu(GeV) nue numu nutau nuebar numubar nutaubar (all fluxes in nus/cm^2/POT/GeV)
+Enu(GeV) nue numu nutau nuebar numubar nutaubar (all fluxes in nus/cm^2/POT/GeV)
```

Comment:
Is removing the `#` comment character okay here?

Reply:
yup! we use the header information when grabbing the flux for each primary type